nuggetsapp/views.py

```diff
@@ -37,5 +40,11 @@ def add_nugget_post(request):
 
 @login_required
 def my_nuggets(request):
-    nuggets = Nugget.get_nuggets_by_user(request.user)
+    nuggets = Nugget.get_nuggets_by_user(request.user.id)
     return render(request, 'nuggetsapp/my-nuggets.html', {'nuggets': nuggets})
+
+@api_view(['GET'])
```

Comment:
I'd suggest that as we start building out more APIs (or we can do this now), that we separate API controllers into it's own file to separate from view controllers.  like an `api_controllers.py` or something.